src/chrome/content/rules/Payu.ru.xml

```diff
@@ -0,0 +1,13 @@
+<!--
+Mismatch on:
+(www.)bpp.payu.ru
+^payu.ru
+-->
+<ruleset name="Payu.ru">
+	<target host="www.payu.ru" />
+	<target host="payu.ru" />
+	<rule from="^http://payu.ru/"
```

Comment:
You need to escape the `.` to `\.`. Just read the errors returned by the Travis tests.